google/cloud/bigquery/datatransfer/v1/BUILD.bazel

```diff
@@ -14,6 +14,9 @@ proto_library(
     ],
     deps = [
         "//google/api:annotations_proto",
+        "//google/api:client_proto",
```

Comment:
Can we add the instructions to updating the BUILD.bazel file in https://docs.google.com/document/d/1loAC6J7ZS2naeTYSixwiOkSzkKM-Z-4zYCPIL84sc4o/edit#?